tests/backends/prometheus.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats
+
+validate_metrics() {
+	fname="${1}"
+	params="${2}"
+
+	if [ ! -f .gitignore ];	then
```

Comment:
this should be in `setup()` function which is executed by bats. Look at `lifecycle.bats`

Reply:
I tried it like that first, but it fails silently when the condition isn't met. No error message and no echo.  It's better like this, you know why it failed.

Reply:
You tried it exactly as it is in https://github.com/netdata/netdata/blob/master/tests/lifecycle.bats#L11-L16 ? I mean with `exit 1` instead of `return 1`?

Reply:
Also this is sth that belongs to environment setup and not to test itself. If we want to have clean code, it still should go to `setup()` and if it doesn't show error msg, then an issue in BATS framework should be created.

Reply:
Moved it to global scope as per 
https://stackoverflow.com/questions/49991275/is-there-a-way-to-bail-from-a-bats-test

Skipping tests from within the setup() function doesn't apply, plus the example with the echo there shows the failure at the echo and then outputs the message, which isn't correct.